Database/Corrections/SeasonOfDiscovery.lua

```diff
@@ -361,6 +363,7 @@ local questsToBlacklistBySoDPhase = {
         [90200] = true, -- Hiding Rogue Mutilate Dun Morogh for now as there are too many icons
         [90201] = true, -- Hiding Rogue Shiv Duskwood for now as there are too many icons
         [90204] = true, -- Hiding Shaman Way of Earth Silverpine Forest for now as there are too many icons
+        [90214] = true, -- Hiding Shaman Water Shield The Barrens for now as there are too many icons
```

Comment:
If we only show the chest locations we do not need to hide this rune.

```suggestion
```

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5610/commits/9df6737c3ece0b93e05f8449b1800e640518c8ee